### PR TITLE
chore: add window eye icon

### PR DESCRIPTION
**Pull request by kark**

### CODE REVIEW

File: packages/components/icons/src/generated/CustomViewsSelectorReact.tsx
Comment (kark) [inline packages/components/icons/src/generated/CustomViewsSelectorReact.tsx:1]: Any suggestions for a more generic name for this icon?

Comment (emmenko) [reply packages/components/icons/src/generated/CustomViewsSelectorReact.tsx:1]: We usually name the icon to what it represents, not for what is meant to be used for.

In this case something like `window-eye.react.svg`

**Review by CarlosCortizasCT — APPROVED** [review]
Many thanks! 👍

**Review by emmenko — APPROVED** [review]

**Review by chloe0592 — APPROVED** [review]
Thank you!

**Pull request merged**